### PR TITLE
Update support contact to point at YourIT

### PR DESCRIPTION
**Pull request by pkarman**
connects #10804 

Screen Shot 2019-06-25 at 3 21 30 PM

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -272,6 +272,8 @@
 
   # :nocov:
   namespace :test do
+    get "/error", to: "users#show_error"
```
Comment (pkarman) [inline config/routes.rb]: just to make it easier to visually test locally

**Comment by pkarman** [comment]
@leikkisa agreed about the COPY.json file -- I will leave that for another PR.

**Comment by pkarman** [comment]
it does not appear that links are used (possible?) in the COPY.json file.

**Comment by leikkisa** [comment]
Yeah, I use a "_START" and "_END" suffix in COPY for links but it is a little awkward, and I generally don't feel strongly about using the COPY file. Some of them don't seem to have links too so maybe just for those.

**Pull request merged**